automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -218,3 +216,12 @@ final class LeonardoTerraDockerSuite
     with TestSuite
     with GPAllocBeforeAndAfterAll
     with ParallelTestExecution
+
+final class LeonardoAppSuite
```

Comment:
Odd this is needed.. I wonder if it increases the test execution time - do they suites run in parallel?

Reply:
I broke this up becuz I vaguely remember gpalloc prematurely releasing project when we chain too many tests in one suite...but it seems like in this case, I just had an extra `GPAllocBeforeAndAfterAll`...if tests pass, think I'll just merge this to catch release. We can have another PR to remove this separate suite and see if things work